python/graphframes/tests.py

```diff
@@ -123,28 +135,38 @@ def setUp(self):
     def test_spark_version_check(self):
         gtu = GraphFrameTestUtils
         gtu.spark_version = gtu.parse_spark_version("2.0.2")
-        self.assertTrue(gtu.spark_at_least_of_version("1.7"))
-        self.assertTrue(gtu.spark_at_least_of_version("2.0"))
-        self.assertTrue(gtu.spark_at_least_of_version("2.0.1"))
-        self.assertTrue(gtu.spark_at_least_of_version("2.0.2"))
-        self.assertFalse(gtu.spark_at_least_of_version("2.0.3"))
-        self.assertFalse(gtu.spark_at_least_of_version("2.1"))
+
+        assert gtu.spark_at_least_of_version("1.7")
+        assert gtu.spark_at_least_of_version("2.0")
+        assert gtu.spark_at_least_of_version("2.0.1")
+        assert gtu.spark_at_least_of_version("2.0.2")
```

Comment:
what is this?

Reply:
@bjornjorgensen I do not know, this is not an addition I made. It is not in my PR.

Reply:
?

Reply:
I converted it from unittest to pytest. I can alter the logic of the tests, but that was not the purpose of this PR.